# Copyright 2020 RUBIS

/**:
  ros__parameters:
    vehicle:
      cg_to_front_m: 1.0
      cg_to_rear_m: 1.0
      front_corner_stiffness: 0.1
      rear_corner_stiffness: 0.1
      mass_kg: 1500.0
      yaw_inertia_kgm2: 12.0
      width_m: 2.0
      front_overhang_m: 0.5
      rear_overhang_m: 0.5
    safety_factor: 1.1
    stop_margin: 12.5  # distance between control point (e.g. CoG/base_link) and obstacle
    trajectory_smoother:
      kernel_std: 5.0
      kernel_size: 25
    staleness_threshold_ms: 500
    target_frame_id: "map"
    lookahead_boxes: 20
